static/js/components/dashboard/CourseRow.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+
+import CourseAction from './CourseAction';
+import CourseGrade from './CourseGrade';
+import CourseDescription from './CourseDescription';
+import type { Course } from '../../flow/programTypes';
+
+export default class CourseRow extends React.Component {
+  props: {
+    course: Course,
+    now: Object
+  };
+
+  render() {
+    const { course, now } = this.props;
+
+    return <Grid className="course-row">
+      <Cell col={6}>
+        <CourseDescription course={course} now={now} />
```

Comment:
looks like you could just splat `{...this.props}` into all these course components here, maybe it's better to be explicit tho?